molecule/default/latest-converge.yml

```diff
@@ -10,6 +10,8 @@
         cache_valid_time: 600
       when: ansible_os_family == 'Debian'
       changed_when: false
-
+  vars:
+    nextcloud_disable_apps:
+      - photos
```

Comment:
Could you add 2 apps to test scenario? Sometimes lists behave a little different for multiple values, so we would catch this misbehavior.

Reply:
I've added theming and mail in my latest commit